docker-compose.yml

```diff
@@ -59,16 +60,17 @@ services:
       - POSTGRES_USER=kinetic
       - POSTGRES_PASSWORD=kinetic
       - POSTGRES_DB=kinetic_test
-  petstore:
-    image: "swaggerapi/petstore3"
-    networks:
-      - kinetic
-    environment:
-      - SWAGGER_HOST=http://0.0.0.0:4008
-      - SWAGGER_BASE_PATH=/api/v1
-      - SWAGGER_URL=http://0.0.0.0:4006
-    ports:
-      - "4008:8080"
+#  TODO: Investigate if needed
+#  petstore:
+#    image: "swaggerapi/petstore3"
+#    networks:
+#      - kinetic
+#    environment:
+#      - SWAGGER_HOST=http://0.0.0.0:4008
+#      - SWAGGER_BASE_PATH=/api/v1
+#      - SWAGGER_URL=http://0.0.0.0:4006
+#    ports:
+#      - "4008:8080"
```

Comment:
`petstore` is just a graphical browser for the open api definitions.  it's sometimes nice to use to debug things, but doesn't need to run all the time.  You could move the definition to a separate compose file and add the directions to the README so it could be started when needed: https://stackoverflow.com/questions/45680958/can-you-define-optional-docker-compose-services